contracts/Token/RaftToken.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.19;
+
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+import { Ownable2Step } from "@openzeppelin/contracts/access/Ownable2Step.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+/// Raft Token with initial supply that be increased by up to 2% yearly after 4 years.
```

Comment:
```suggestion
/// Raft Token with initial supply that be increased by up to 2% yearly after every 4 years.
```

Reply:
after 2.5 years, mint can happen yearly, I think this is good

Reply:
Yeah, you are right.